packages/opentelemetry-core/src/trace/NoRecordingSpan.ts

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import {
-  SpanContext,
-  NoopSpan,
-  INVALID_SPAN_CONTEXT,
-} from '@opentelemetry/api';
-
-/**
- * The NoRecordingSpan extends the {@link NoopSpan}, making all operations no-op
- * except context propagation.
- */
-export class NoRecordingSpan extends NoopSpan {
```

Comment:
Breaking change, yes?

Reply:
For core yes but not for API. Not sure if we mark PRs not effecting API as breaking.

Reply:
we typically do since versions everywhere move in lockstep for now. when API goes to 1.0 this will change.

Reply:
label added